src/libs/AxelarAssetTransfer.ts

```diff
@@ -266,6 +294,104 @@ export class AxelarAssetTransfer {
     return address.toLowerCase();
   }
 
+  /**
+   * @param {Object}  requestParams
+   * @param {string}  requestParams.fromChain - Source chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.toChain - Destination chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.destinationAddress - Address where the asset should be transferred to on the destination chain
+   * @param {Object}  requestParams.options
+   */
+  public async sendToken(requestParams: SendTokenParams) {
+    const { fromChain, toChain } = requestParams;
+
+    await throwIfInvalidChainIds([fromChain, toChain], this.environment);
+
+    const chainList: ChainInfo[] = await loadChains({ environment: this.environment });
+    const srcChainInfo = chainList.find((ch) => ch.id === fromChain.toLowerCase());
+    if (!srcChainInfo) throw new Error("cannot find chain" + fromChain);
+
+    return srcChainInfo.module === "evm"
+      ? this.sendTokenFromEvmChain(requestParams)
+      : this.sendTokenFromCosmosChain(requestParams, srcChainInfo);
+  }
+
+  private async sendTokenFromEvmChain(requestParams: SendTokenParams) {
+    //todo
+  }
+
+  private async sendTokenFromCosmosChain(requestParams: SendTokenParams, chain: ChainInfo) {
+    if (!requestParams.options?.cosmosSendTokenOptions) throw `need a cosmos signer`;
+    const {
+      fromChain,
+      toChain: destination_chain,
+      destinationAddress: destination_address,
+      options: {
+        cosmosSendTokenOptions: {
+          cosmosDirectSigner,
+          rpcUrl,
+          transferAmount,
+          timeoutHeight,
+          timeoutTimestamp,
+          fee,
+        },
+      },
+    } = requestParams;
+    if (fromChain === CHAINS[this.environment.toUpperCase() as "TESTNET" | "MAINNET"].AXELAR)
+      throw `sending cross-chain using sendToken directly from Axelar network is currently not supported`;
+
+    const memo = JSON.stringify({
+      destination_chain,
+      destination_address,
+      payload: null,
+      type: 3, // type 3 corresponds to the `sendToken` command on Axelar
+    });
+
+    const signingClient = await SigningStargateClient.connectWithSigner(rpcUrl, cosmosDirectSigner);
+    const { address: senderAddress } = (await cosmosDirectSigner.getAccounts())[0];
+
+    const AXELAR_GMP_ACCOUNT_ADDRESS =
+      "axelar1dv4u5k73pzqrxlzujxg3qp8kvc3pje7jtdvu72npnt5zhq05ejcsn5qme5";
+
+    const typeUrl = "/ibc.applications.transfer.v1.MsgTransfer";
+    const value = MsgTransfer.fromPartial({
+      sender: senderAddress,
+      receiver: AXELAR_GMP_ACCOUNT_ADDRESS,
+      token: transferAmount,
```

Comment:
`transferAmount` also includes the `IBC denom` on that chain which consumers of this API might not want to deal with (for e.g. they pass in "wavax-wei", and we figure out the the IBC denom to use). Having said that, most apps should just get the memo field and use their existing logic to handle signing I think.

Reply:
Included transferAmount in the signerOptions, but to the point, should move into the main parameters, so will do that